src/autoupdater.ts

```diff
@@ -404,13 +409,33 @@ export class AutoUpdater {
         break;
       } catch (e: unknown) {
         if (e instanceof Error) {
+          /**
+           * If this update was against a fork and we got a 403 then it's
+           * probably because we don't have access to it.
+           */
+          if (
+            'status' in e &&
+            (e as RequestError).status === 403 &&
+            sourceEventOwner !== mergeOpts.owner
+          ) {
+            ghCore.error(
```

Comment:
do you think it would be useful to include github's actual error message here as well?

Reply:
Might be too verbose for the annotation? Though, I'll add it in as a debug message, might be helpful for figuring out issues

Reply:
It's not too bad tbh, what do you think? https://github.com/chinthakagodawita/autoupdate-test/actions/runs/1181370405

Reply:
Yeah I think that's good